content/support.md

```diff
@@ -1,4 +1,8 @@
 +++
 title = "Support"
-description = "TBD"
 +++
+
+## Need help with BotKube? 
+
+Reach us out at *kubeops@infracloud.io*
```

Comment:
Email is correct ? Should it be `botkube@infracloud.io`

Reply:
Fixed and pushed to master